Misha/OB-159_reverse_a_string/string_reverse.py

```diff
@@ -0,0 +1,20 @@
+def string_reverser(string):
+    if string is None:
+        raise TypeError('Non-type value')
+    else:
+        reversed_string = ''
+        string_size = len(string)
+        print('string to reverse: [' + string + ']')
+        for index in range(string_size):
+            reversed_string += string[string_size - index - 1]
+        print('reversed string: [' + reversed_string + ']')
+        return reversed_string
```

Comment:
Good job! Solution is testes and looks like work correctly! But could you do it in one line please? :)

Reply:
i thought i should do it manually, not with in-built functions